runners/direct-java/src/main/java/org/apache/beam/runners/direct/WriteWithShardingFactory.java

```diff
@@ -60,9 +60,10 @@
   public PTransformReplacement<PCollection<InputT>, PDone> getReplacementTransform(
       AppliedPTransform<PCollection<InputT>, PDone, PTransform<PCollection<InputT>, PDone>>
           transform) {
-
     try {
-      WriteFiles<InputT> replacement = WriteFiles.to(WriteFilesTranslation.getSink(transform));
+      WriteFiles<InputT, ?, ?> replacement = WriteFiles.to(
+          WriteFilesTranslation.getSink(transform),
```

Comment:
This code was already here. @kennknowles can comment, but I think the intent is that the override code here will be done via protos, and this code is prep for that